packages/tools/devtools/devtools-view/src/components/TelemetryView.tsx

```diff
@@ -172,6 +173,28 @@ export function TelemetryView(): React.ReactElement {
 		usageLogger?.sendTelemetryEvent({ eventName: "RefreshTelemetryButtonClicked" });
 	};
 
+	const downloadTelemetryLog = (): void => {
+		if (!telemetryEvents) return;
```

Comment:
If there are no telemetry logs available, I'd expect there should be some type of feedback for the user. Maybe some text/ popup saying "No telemetry events available for download."